lib/LaTeXML/Post/Manifest/Epub.pm

```diff
@@ -208,12 +222,25 @@ sub process {
       my $itemref = $spine->addNewChild(undef, 'itemref');
       $itemref->setAttribute('idref', $item_id);
 
-      # Add to navigation
-      my $nav_map = $$self{nav_map};
-      my $nav_li  = $nav_map->addNewChild(undef, 'li');
-      my $nav_a   = $nav_li->addNewChild(undef, 'a');
-      $nav_a->setAttribute('href', URI::file->new($relative_destination));
-      $nav_a->appendText($file); } }
+      # Add to default navigation document
+      if (!$nav_xhtml && !$tocfound) {
+        my $nav_li = $nav_map->addNewChild(undef, 'li');
+        my $nav_a  = $nav_li->addNewChild(undef, 'a');
+        $nav_a->setAttribute('href', URI::file->new($relative_destination));
+        my ($title) = split(/ \x{2023} /, $doc->findnode('/xhtml:html/xhtml:head/xhtml:title')->textContent, 2);
```

Comment:
i suspect this may be a bit more reliable to build from the CrossRef-provided metadata, but I'll let @brucemiller brainstorm that. If the `\x{2023}` delimiter, or its spacing, changes down the road, the regex will fail, which is a bit tricky.

Reply:
Where do I pull out the CrossRef data? I could only find a tag + text in the db, and it seemed like content supposed to go through XSLT. `<xhtml:title>` is already postprocessed, so it seemed like an easier option. That is to say, I couldn't figure out the correct source for the titles!

Talking about `<xhtml:title>`, we might as well simplify the tag in the EPUB3 stylesheet. The titles don't need to contain the document title and all the parent sections, we are not in a browser in this case.

(If the regex fails, `$title` will just be the full string, AFAIK.)

Reply:
CrossRef fills in an `ltx:TOC` with data, so it may be that it's already there, or could easily be, in some form.
I haven't yet looked at your PR to see exactly what you're after with "custom" TOC. 

Commenting a bit further back: my strategy has been that XSLT isn't creating files, but transforming them; all the paths should already be embedded somewhere as relative paths.

Reply:
Re `<xhtml:title>`, I have adopted a more sensible strategy (forced pushed also in #1643 because that part could be mergeable as it is). I simply removed the context from the HTML title (using a convenient parameter that was already there) and cloned the title into the navigation document. Having navigation entry == HTML document title is simple to reason about, I think, easy to customize for the user, etc.